components/TicketsSection.js

```diff
@@ -67,23 +69,29 @@ function TicketsRow ({ title, date, isFull }) {
   )
 }
 
-function Button ({ children, href }) {
+function Button ({ children, href, date }) {
   return (
-    <a className='button' href={href}>
+    <a className={`button ${checkTicket(date) ? 'buynow' : 'waitbuy'}`} href={href}>
       {children}
       <style jsx>{`
         .button {
           display: inline-block;
           font-size: 18px;
           line-height: 33px;
-          color: #00D8FF;
-          border: 2px solid #00D8FF;
           padding: 0 30px;
           border-radius: 35px;
           text-transform: uppercase;
           text-decoration: none;
           transition: 0.15s all ease-out;
         }
+        .waitbuy {
```

Comment:
The button shouldn’t know about what it’s being used for. This couples a generic UI component to a very specific use-case, which makes the button hard to re-use.

A better solution is to use more generic names such as `accent` (for light blue) and `highlighted` (for pink).